CIFAR10/train/fastai_pytorch.tsv

```diff
@@ -0,0 +1,51 @@
+epoch	hours	top1Accuracy
+0	0.0022222222222222222	14.78
+1	0.003611111111111111	24.88
+2	0.004722222222222222	36.3
+3	0.006111111111111111	31.98
+4	0.007222222222222222	57.94
+5	0.008333333333333333	58.09
+6	0.009722222222222222	68.69
+7	0.010833333333333334	68.88
+8	0.012222222222222223	70.78
+9	0.013333333333333334	76.04
+10	0.014722222222222222	75.35
+11	0.015833333333333335	77.55
+12	0.017222222222222222	78.21
+13	0.018333333333333333	74.08
+14	0.019444444444444445	73.65
+15	0.020833333333333332	70.57
+16	0.021944444444444444	78.31
+17	0.023055555555555555	80.58
+18	0.024166666666666666	80.83
+19	0.025555555555555557	78.52
+20	0.02666666666666667	78.97
+21	0.028055555555555556	73.26
+22	0.029166666666666667	80.28
+23	0.030555555555555555	74.13
+24	0.03194444444444444	72.7
+25	0.03305555555555555	83.4
+26	0.034166666666666665	73.58
+27	0.035555555555555556	82.86
+28	0.03666666666666667	81.83
+29	0.03777777777777778	87.83
+30	0.03916666666666667	85.89
+31	0.04027777777777778	81.5
+32	0.04138888888888889	85.28
+33	0.042777777777777776	85.89
+34	0.04388888888888889	86.62
+35	0.04527777777777778	86.13
+36	0.04638888888888889	88.77
+37	0.04777777777777778	88.92
+38	0.049166666666666664	90.41
+39	0.050277777777777775	91.95
+40	0.05138888888888889	92.87
+41	0.05277777777777778	93.18
+42	0.05388888888888889	94.01
+43	0.05527777777777778	94.04
+44	0.05638888888888889	94.16
+45	0.0575	94.04
+46	0.058888888888888886	94.25
+47	0.06	94.23
+48	0.06111111111111111	94.26
+49	0.0625	94.21
```

Comment:
These timestamps do not seem very precise. Can you update your timer to at least millisecond precision?

Reply:
switch to millisecond timing and re-ran the script!